src/common.rs

```diff
@@ -497,20 +497,20 @@ impl<'a> IntoIterator for &'a Networks {
     }
 }
 
-/// Enum containing the different supported disks types.
+/// Enum containing the different supported kinds of disks.
 ///
-/// This type is returned by [`Disk::get_type`][crate::Disk#method.type].
+/// This type is returned by [`crate::DiskExt::kind`].
```

Comment:
It'll be displayed as `crate::DiskExt::kind`, which isn't great. Can you use the old format please?